command/exec.go

```diff
@@ -187,6 +191,14 @@ func (c *subprocessExec) Execute(ctx context.Context, comm client.Communicator,
 		return errors.WithStack(err)
 	}
 
+	if !c.NoStripBlankArgs {
+		for i, arg := range c.Args {
+			if arg == "" {
+				c.Args = append(c.Args[:i], c.Args[i+1:]...)
```

Comment:
My intuition is that this shouldn't work because you're modifying the thing you're iterating over, but I haven't been able to prove otherwise.

Reply:
https://golang.org/ref/spec#RangeClause